pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -223,3 +253,13 @@ func (c *Check) GetObfuscatedStatement(o *obfuscate.Obfuscator, statement string
 func (c *Check) getFullPDBName(pdb string) string {
 	return fmt.Sprintf("%s.%s", c.cdbName, pdb)
 }
+
+func (c *Check) getTagsWithPDB(pdb sql.NullString) []string {
+	var pdbName string
+	if pdb.Valid {
+		pdbName = pdb.String
+	} else {
+		pdbName = ""
+	}
+	return append(c.tags, "pdb:"+pdbName)
```

Comment:
Don't we want to just leave out the pdb tag if the value isn't valid (rather than putting a blank `pdf:` tag)?

Reply:
I can leave it out. Will change it tomorrow.

Reply:
done